packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -43,21 +43,27 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
   public render() {
     const {
       calloutProps,
-      tooltipProps,
-      content,
       children,
+      content,
+      delay,
       directionalHint,
       directionalHintForRTL,
-      delay,
+      hostClassName,
       id,
+      overflowMode,
       setAriaDescribedBy = true,
-      hostClassName
+      tooltipProps
     } = this.props;
     const { isTooltipVisible } = this.state;
     const tooltipId = id || getId('tooltip');
+
     return (
       <div
-        className={ css('ms-TooltipHost', styles.host, hostClassName) }
+        className={ css('ms-TooltipHost',
+          styles.host,
+          hostClassName,
+          overflowMode !== undefined && styles.hostOverflow
```

Comment:
Hmm... this seems unnatural. if it is defined, even set to false or 0, this works?

Reply:
If set to 0, overflowmode is parent and we want it to have the hostOverflow style. If defined, we assume that overflowmode is either parent or self and we want it to have the hostOverflow style.